editor/src/components/canvas/controls/select-mode/select-mode-hooks.ts

```diff
@@ -257,10 +257,6 @@ function useStartDragState(): (
         entireEditorStoreRef.current.editor,
       )
 
-      const elementsThatRespectLayout = selectElementsThatRespectLayout(
```

Comment:
`selectElementsThatRespectLayout` prevents moving/resizing elements that can't accept style layout props, dragging should keep them in place to show this behavior. Resizing also uses this helper somewhere near here to show the resize corner on elements that can be really resized.

Reply:
Seems like the next item on the track is to fix that exact function, so I'll reintroduce this call with that work, just at the moment it's filtering out everything that isn't from the canvas file by default.

Reply:
could you leave a TODO here please?

also, are there any other known regressions made by this PR? it'd be worth tracking them in the paper doc before we loose track of some…

Reply:
Not known ones no.

Reply:
> Why has this filtering been removed?
Because for the dev project it held just two values, which were the same thing twice and that resulted in effectively nothing being movable or resizable.

Reply:
Which is by design since the dev project doesn't define any property controls, which are what's used to determine if an element can be dragged.

Reply:
Except when I say that I mean even when the Card is focused and there are `Rectangle` elements in the mix.

Reply:
Yep which appears to be a bug on master that affects a focused component imported from another file, which should either be fixed separately, or the filtering removed as a conscious decision in a separate PR. Otherwise this is mixing an unexpected feature change into this PR.

Obviously the bug got in the way of developing this feature, but I'd rather we didn't remove the filtering. We should either provide some (strictly temporary) way to disable it (for the sake of this work and passing the tests) and address the bug immediately after, or discuss removing the filtering completely.

Reply:
See this comment: https://github.com/concrete-utopia/utopia/pull/1088#discussion_r614031198

Reply:
I'm good with a FIXME + tracking this

Reply:
Added in c02ad628.